### PR TITLE
task iV2vejQJ

### PR DESCRIPTION
**Pull request by BYavorskyj**
Add a new component - Footer
https://trello.com/c/iV2vejQJ/6-add-footer-component

### CODE REVIEW

File: src/client/App.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import './app.css';
 import ReactImage from './react.png';
+import Footer from './components/Footer';
```
Comment (vkryviak-ua) [inline src/client/App.js]: Where is a Footer.js file ?

File: src/client/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+
+
+export default class App extends Component {
```
Comment (vkryviak-ua) [inline src/client/components/Footer.js]: Incorrect name

Comment (BYavorskyj) [reply src/client/components/Footer.js]: changed and updated

File: src/client/components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+
+
+export default class Footer extends Component {
```
Comment (AntonShalenui) [inline src/client/components/Footer.js]: you are lushpak

**Pull request merged**